src/apis/guids.ts

```
@@ -0,0 +1,17 @@
+const guides = 'guides';
+
+export const getGuides = async () => {
+  try {
+    const response = await instance.get(`${guides}`, {
+      headers: {
+        Authorization: `Bearer ${await getItem('accessToken')}`
+      }
+    })
+    return response.data;
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)